_includes/snippets_library/UCL_Myriad_sge/modules/wrong-gcc-version.snip

```diff
@@ -7,7 +7,7 @@ e.g.
 {: .language-bash}
  
 ```
-------------------------------------- /shared/ucl/apps/modulefiles/applications --------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~ /shared/ucl/apps/modulefiles/applications ~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This is a complete hack but I'm not willing to spend more time trying to debug `lesson_check.py`